newrelic-php-agent/values.yaml

```diff
@@ -1,11 +1,41 @@
-# Default values for simple-echo.
+# Default values for newrelic-php-agent.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-image:
-  repository: chatwork/newrelic-php-agent
-  tag: latest
-  pullPolicy: IfNotPresent
-  command: []
-
-nameOverride: ""
-fullnameOverride: ""
\ No newline at end of file
+newrelic:
+  image:
+    repository: chatwork/newrelic-php-agent
+    tag: 0.0.1
+    pullPolicy: IfNotPresent
+    command: []
+
+  livenessProbe:
+    tcpSocket:
+      port: 31339
+    failureThreshold: 2
+    initialDelaySeconds: 10
+    periodSeconds: 60
+
+  readinessProbe:
+    tcpSocket:
+      port: 31339
+    failureThreshold: 2
+    periodSeconds: 5
+
+  socketfile:
+    enabled: true
+
+  daemonset:
+    enabled: true
+
+  deployment:
+    enabled: false
+    replicaCount: 1
+
+  extraPorts: []
+  extraEnv: []
+  extraEnvFrom: []
+  extraVolumes: []
+  extraVolumeMounts: []
```

Comment:
@cw-niinuma 
Please tell me the use case of `extraXXX`.
Also, please tell me why you can use it in `deployment` even though it can't be used in `daemonset`.